packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -467,6 +467,7 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
       peerList: [...new Set([localAddress, ...metadata.peers])],
       psk: metadata.psk,
       ownerOrbitDbIdentity: metadata.ownerOrbitDbIdentity,
+      inviteData,
```

Comment:
I would add a comment here to remind us that current users will not have this field unless we do a migration